.idea/gradle.xml

```diff
@@ -11,6 +11,7 @@
           <set>
             <option value="$PROJECT_DIR$" />
             <option value="$PROJECT_DIR$/example" />
+            <option value="$PROJECT_DIR$/mylibrary" />
```

Comment:
what is this `mylibrary` option?

Reply:
Oh Sorry, this was accidently added by me while I was trying some other thing. I've removed it now.